frontend/app/src/main/java/com/chads/vanroomies/Constants.java

```diff
@@ -2,7 +2,8 @@
 
 public class Constants {
     // Endpoints
-    final static String baseServerURL = "https://20.121.136.167:3000";
+    // TESTING: final static String baseServerURL = "https://10.0.2.2:3000";
+    final static String baseServerURL = "https://20.121.136.167:3000"; // TESTING: "https://10.0.2.2:3000"
```

Comment:
Can we get rid of the comment beside this line hehe since we already have the comment above this line?

Reply:
oh oops! Thanks!